docs/install.rst

```diff
@@ -5,58 +5,112 @@ Installation
+
+   .. code-block:: shell
+
+      # create the environment
+      $ python -m venv path/to/environments/plenoptic-venv
+      # activate the environment
+      $ source path/to/environment/plenoptic-venv/bin/activate
+      $ pip install plenoptic
+
+.. tab:: Windows
+
+   .. code-block:: powershell
+
+      # create the environment
+      $ python -m venv path\to\environments\plenoptic-venv
+      # activate the environment
+      $ path\to\environment\plenoptic-venv\bin\activate
+      $ pip install plenoptic
+
+Note that when using ``venv``, you have to decide where you'd like to place the folder containing all files related to the virtual environment. If the virtual environment is related to the development of a package, as in :ref:`source`, it is typical to place them within the repository for that package. Otherwise, it is typical to place them all in a single directory somewhere so they're easy to keep track of.
+
+.. _source:
+
+Installing from source
+^^^^^^^^^^^^^^^^^^^^^^
+
+You can also install plenoptic directly from source in order to have a local editable copy. This is most useful for developing (for more info, see `our contributing guide <https://github.com/LabForComputationalVision/plenoptic/blob/main/CONTRIBUTING.md>`_) or if you want to use the most cutting-edge version.
+
+.. tab:: Linux / MacOS
+
+   .. code-block:: shell
+
+      $ git clone https://github.com/LabForComputationalVision/plenoptic.git
+      $ cd plenoptic
+      # create the environment
+      $ python -m venv .venv
+      # activate the environment
+      $ source .venv/bin/activate
+      # install in editable mode with `-e` or, equivalently, `--editable`
+      $ pip install -e .
+
+.. tab:: Windows
+
+   .. code-block:: powershell
+
+      $ git clone https://github.com/LabForComputationalVision/plenoptic.git
+      $ cd plenoptic
+      # create the environment
+      $ python -m venv .venv
+      # activate the environment
+      $ .venv\bin\activate
+      # install in editable mode with `-e` or, equivalently, `--editable`
+      $ pip install -e .
 
 With an editable copy, any changes locally will be automatically reflected in your installation (under the hood, this command uses symlinks).
 
+Note that, with the above setup, all files related to your virtual environment are stored in a hidden directory named ``.venv`` within the ``plenoptic/`` directory you cloned. Therefore, if you delete the ``plenoptic/`` directory, you'll need to rerun the setup above to create a new virtual environment.
+
 .. attention:: To install ``plenoptic`` in editable mode, you need ``pip >= 21.3`` (see pip's `changelog <https://pip.pypa.io/en/stable/news/#id286>`_). If you run into `an error <https://github.com/LabForComputationalVision/plenoptic/issues/227>`_ after running the ``pip install -e .`` command, try updating your pip version with ``pip install --upgrade pip``.
 
 .. _optional-deps:
+
 Optional dependencies
----------------------
+^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This should probably be a sub-section of the installing from source;

Reply:
I think it's not, because you could also use it to setup the environment for running jupyter notebooks (see the jupyter section farther down).